library/core/tests/num/wrapping.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(arithmetic_overflow)] // Overflowing is part of the test.
 use core::num::Wrapping;
```

Comment:
Is MIR inlining triggrring new warnings? I would have thought that they would be suppressed